packages/remix-dev/__tests__/create-test.ts

```diff
@@ -391,6 +391,22 @@ describe("the create command", () => {
     expect(fse.existsSync(path.join(projectDir, "remix.init"))).toBeFalsy();
   });
 
+  it("runs remix.init script when using index.ts", async () => {
+    let projectDir = await getProjectDir("remix-init-auto");
+    jest.setTimeout(100000);
+    await run([
+      "create",
+      projectDir,
+      "--template",
+      path.join(__dirname, "fixtures", "stack-init-ts.tar.gz"),
```

Comment:
I just copied the stack under `/fixtures/stack`, converting the index file to `index.ts` and console logging the message below in the default method exported